docs/data/material/getting-started/learn/learn.md

```diff
@@ -1,61 +1,53 @@
-# Learn MUI
+# Learning resources
 
-<p class="description">New to MUI? It's easy to learn if you know where to start! This guide will help you to get started quickly.</p>
+<p class="description">New to Material UI? Get up to speed quickly with our curated list of learning resources.</p>
 
-Developers come to MUI from different backgrounds and with different learning styles. Whether you prefer a more theoretical or practical approach, we hope you'll find this section helpful.
-Like any unfamiliar technology, MUI does have a learning curve. With practice and some patience, you will soon get the hang of it.
+## Your first component
 
-## First example
+The following code snippet demonstrates a basic Material UI app that features a `<Button>` component:
 
-The [usage page](/getting-started/usage/#quick-start) contains a small MUI example with a live editor.
-Even if you don't know anything about MUI yet, try changing the code and see how it affects the result.
+```jsx
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '@mui/material/Button';
 
-## Example projects
+function App() {
+  return (
+    <Button variant="contained" color="primary">
+      Hello World
+    </Button>
+  );
+}
 
-There are [some example projects](/getting-started/example-projects/) available, providing the infrastructure needed to develop and deploy websites in React.
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
 
-## Templates
+In the interactive demo below, try changing the code and see how it affects the output. (Hint: change `variant` to `"outlined"` and `color` to `"secondary"`. For more options, see the [`Button` component page](components/buttons/).)
 
-This [selection of basic templates](/getting-started/templates/) will help you get started building your app.
+{{"demo": "../usage/Usage.js", "hideToolbar": true, "bg": true}}
```

Comment:
**Off-topic**.

This .. sounds like a good signal that restoring absolute path imports would be better.

Reply:
Do you have a discussion opened about this topic ?
Because in most demos, the new relative path is much simpler to play with that the absolute path we had before.

Reply:
Interesting feedback. I can only recall another comment I left on a PR of Jun but I never followed what was the answers 😁. The problem I envision is about being able to quickly find a demo by looking at the markdown. Before, I could open the markdown, and copy and paste the path in my IDE, it would at 100% return the right file. Now, we have relative paths and non-unique demo files. It's trickier to open the right file.

I have no visibility on the upside of this new approach, I also so the cons, I guess it's a tradeoff then.

Reply:
On X the demo is always on the same demo as the markdown
So it is the opposite, the relative path makes it very obvious and help me find the demo quicker.

Reply:
No preference on my side. What relative path really benefits me is the migration when I move them into different folders (it just works).

> quickly find a demo source by looking at the markdown

Usually, I copy the last part of the demo and search. I haven't encountered the wrong file before because most demos have a unique name based on the folder/component name.

Reply:
It sounds like the pain I'm referring to doesn't resonate, fair enough. I think that we can pause it here. I leave this comment unresolved for visibility in case it comes up in the future.